submissions/sayopaul/Week1/fizz.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+	for($i=1; $i<=100; $i++){
+
+		$output =  ($i % 15 === 0) ? "FizzBuzz".PHP_EOL  :  ( ($i % 3 === 0)  ? "Fizz".PHP_EOL :  ( ($i % 5 === 0 ) ? "Buzz".PHP_EOL : $i.PHP_EOL ));
```

Comment:
Would be nice to keep just one of these two. We believe code should not be commented if it's good code. You can keep the human readable version and delete the one above.